website/blog/2020-08-31-63updates.md

```diff
@@ -0,0 +1,114 @@
+
+
+# Developer Experience Improvements
+With the new 0.63 version we'll be taking advantage of a few really great improvements to the developer experience that were added by the React Native core team.
+
+## LogBox
+
+![logbox-windows](assets/logbox_demo.gif)
+
+LogBox is now on by default for React Native apps on Windows! This is a huge step forward in addressing some of the pain points experienced by developers when trying to debug their apps.
+
+With this new update you can now more clearly read errors in a concise, formatted, and actionable layout that's easy on the eyes and the mind - a win-win!
+
+Check out the [official blog post](https://reactnative.dev/blog/2020/07/06/version-0.63#logbox) for more information around this feature.
+
+## Autolinking
+
+![platformcolor-windows](assets/autolinking-windows.png)
+
+Sometimes it really is the little things that make a huge difference and in this case Autolinking is no exception!
+
+Now when you install new modules and build your app we will automatically hook-up all those necessary native platform-specific code pieces for you in Windows. This will save you time and (possibly) a headache, if C++ isn't your strong suit.
+
+We've also provided some new documentation around how to use [Autolinking with Native Modules](https://microsoft.github.io/react-native-windows/docs/native-modules-autolinking) in Windows.
+
+# PlatformColor
+
+![platformcolor-windows](assets/platformcolor_demo.gif)
+
+`PlatformColor` is a new API React Native core introduced that gives React Native app components the ability to leverage native system colors - meaning you can now subscribe to certain colors or brushes in your `StyleSheet` that respond to native platform actions like theme switching, user accent colors, or high contrast changes.
+
+Head over to the [official blog post](https://reactnative.dev/blog/2020/07/06/version-0.63#native-colors-platformcolor-dynamiccolorios) about this API to learn more about accessing native colors across all platforms, or check out our [new guidance documentation on PlatformColor](https://microsoft.github.io/react-native-windows/docs/windowsbrush-and-theme) to learn how to take advantage of Windows-specific colors and materials in your app.
+
+> Since the release of this API, the older Windows `AppTheme` API will be deprecated in favor of `PlatformColor`.
+
+# Pressable
+
+[Pressable](https://reactnative.dev/docs/pressable) is now available on React Native for Windows apps as well. This new core API helps make customizing your interaction experiences more robust and easier to fit around your custom components and scenario needs.
+
+# Community Modules
+
+As with every big release, we're still working hard on getting important community modules supported on Windows.

```

Comment:
I've been trying to update my vernacular to swap "supported" with "implemented" to help ensure we're sending an accurate message. 

```suggestion
As with every big release, we're still working hard at adding Windows implementations to important community modules.

```